### PR TITLE
Add new volunteer Abdlrahman

### PR DESCRIPTION
**Pull request by AbdlrahmanSaberAbdo**

### CODE REVIEW

File: data/users/AbdlrahmanSaberAbdo.json [hunk 1]
```diff
@@ -0,0 +1,22 @@
+{
+    "name": "Abdlrahman Saber",
+    "bio": "Software engineer",
+    "image": "https://avatars.githubusercontent.com/u/29666390?v=4",
+    "categories": [
+      {
+        "slug": "web-development",
+        "topics": ["javascript", "typescript", "laravel", "nestJs", "vueJs"]
```
Comment (codewithvoid) [inline data/users/AbdlrahmanSaberAbdo.json:8]: keep tags max 2-3

File: data/users/AbdlrahmanSaberAbdo.json [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+      },
+      {
+        "slug": "programming",
+        "topics": ["python", "php", "javascript", "typescript"]
+      },
+      {
+        "slug": "programming-topics",
```
Comment (codewithvoid) [inline data/users/AbdlrahmanSaberAbdo.json:15]: we don't support these topics yet so remove them for now..will add soon :)

File: data/users/AbdlrahmanSaberAbdo.json [hunk 3]
```diff
@@ -0,0 +1,22 @@
+{
+    "name": "Abdlrahman Saber",
+    "bio": "Software engineer",
```
Comment (codewithvoid) [inline data/users/AbdlrahmanSaberAbdo.json:3]: fix casing: Software Engineer

**Comment by codewithvoid** [comment]
Thanks for the PR @AbdlrahmanSaberAbdo AbdlrahmanSaberAbdo

File: data/users/AbdlrahmanSaberAbdo.json [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+    "image": "https://avatars.githubusercontent.com/u/29666390?v=4",
+    "categories": [
+      {
+        "slug": "web-development",
+        "topics": ["typescript", "laravel", "nestJs"]
```
Comment (codewithvoid) [inline data/users/AbdlrahmanSaberAbdo.json:8]: Can drop TypeScript from here as already mentioned in programming

Comment (codewithvoid) [inline data/users/AbdlrahmanSaberAbdo.json:8]: nit: nestJs -> NestJS

File: data/users/AbdlrahmanSaberAbdo.json (hunk 3 above)
Comment (codewithvoid) [reply data/users/AbdlrahmanSaberAbdo.json:3]: casing is not yet fixed

File: data/users/AbdlrahmanSaberAbdo.json (hunk 4 above)
Comment (AbdlrahmanSaberAbdo) [reply data/users/AbdlrahmanSaberAbdo.json]: should we use camelCase here?

Comment (AbdlrahmanSaberAbdo) [reply data/users/AbdlrahmanSaberAbdo.json]: I took a look for other files some people use camelCase and some people do not! for example, some of them wrote javascript like this `javascript` and others wrote it like this `JavaScript`?

Comment (codewithvoid) [reply data/users/AbdlrahmanSaberAbdo.json]: Yes the naming in some files is not consistent. We are moving to new convention.

Comment (AbdlrahmanSaberAbdo) [reply data/users/AbdlrahmanSaberAbdo.json]: So, what it should look like? camelCase?

Comment (codewithvoid) [reply data/users/AbdlrahmanSaberAbdo.json]: You can update it as following:
```
 "topics": ["VueJS", "laravel", "NestJS"]
```

Comment (AbdlrahmanSaberAbdo) [reply data/users/AbdlrahmanSaberAbdo.json]: Great, but also I think we should update also `laravel` to `Laravel` if we'll use `Pascal case`

Comment (codewithvoid) [reply data/users/AbdlrahmanSaberAbdo.json]: Yes sounds good. Please update.

**Comment by codewithvoid** [comment]
Thanks for making the changes!

**Review by codewithvoid — APPROVED** [review]

**Pull request merged**